README.md

```diff
@@ -34,7 +34,7 @@ Contributions to Absolute are welcomed and encouraged! Please see the
 [Contributing to Absolute guide](./contribution/contribution.md)
 
 ### Building Absolute
-./absolut start
+./absolute start
```

Comment:
lgtm with nit
I'd like to wrap the command into code block[1]. WDYT?
[1] https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code